Sources/TensorFlow/Operators/Comparison.swift

```diff
@@ -210,6 +210,7 @@ public extension Tensor where Scalar: Equatable {
 public extension Tensor where Scalar: TensorFlowFloatingPoint & Equatable {
     /// Returns a tensor of Boolean values indicating whether the elements of `self` are
     /// approximately equal to those of `other`.
+    /// `self` and `other` must be of the same shape.
```

Comment:
```suggestion
    /// - Precondition: `self` and `other` must be of the same shape.
```
Add precondition comments as a `- Precondition:` doc item so that they get better rendered.